src/layouts/Workspace.jsx

```diff
@@ -136,30 +136,16 @@ const Workspace = ({ match, location }) => {
               <div className={contentStyles.segmentDividerContainer}>
                 <hr className="w-100 mt-3 mb-5" />
               </div>
-              {/* Collections title */}
+              {/* Folders title */}
               <div className={contentStyles.segment}>
-                Collections
+                Folders
               </div>
-              {
-                !collections &&
-                <div className={contentStyles.segment}>
-                  Loading Collections...
-                </div>
-              }
-              {
-                collections && collections.length === 0 &&
-                <div className={contentStyles.segment}>
-                  There are no collections in this repository.
-                </div>
-              }
-              {/* Collections */}
+              {/* Folders */}
               <div className={contentStyles.folderContainerBoxes}>
                 <div className={contentStyles.boxesContainer}>
+                  <FolderOptionButton title="Create new folder" option="create-sub" isSubfolder={false} onClick={() => setIsFolderCreationActive(true)}/>
```

Comment:
Could we block this option from appearing until collections have been loaded? Otherwise an error occurs if users attempt to click it too early